### DISCUSSION TITLE
Workout (run) from zwift app on android tablet to treadmill via qz on rpi

### DISCUSSION THREAD

**Discussion by grigory707** [cagnulein/qdomyos-zwift#562] [Q&A]
Hi, Roberto.
Can i create the workout on zwift app and then it send commands to treadmill? Its more usefull then create workout on .xml file on raspberry.

**Comment by cagnulein** [comment]
Yes you can use the zwo file! It's already compatible with QZ!

Il giorno sab 8 gen 2022 alle 13:12 grigory707 ***@***.***>
ha scritto:

> Hi, Roberto.
> Can i create the workout on zwift app and then it send commands to
> treadmill? Its more usefull then create workout on .xml file on raspberry.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/cagnulein/qdomyos-zwift/discussions/562>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAALYWBTX5JNTFYZJEWG67DUVASZXANCNFSM5LQRUHOA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud

**Reply by grigory707** [reply]
Yes. But when i start the workout treadmill don't change speed.

**Reply by grigory707** [reply]
> Yes you can use the zwo file! It's already compatible with QZ! Il giorno sab 8 gen 2022 alle 13:12 grigory707 ***@***.***> ha scritto:
> Hi, Roberto. Can i create the workout on zwift app and then it send commands to treadmill? Its more usefull then create workout on .xml file on raspberry. — Reply to this email directly, view it on GitHub <#562>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAALYWBTX5JNTFYZJEWG67DUVASZXANCNFSM5LQRUHOA> . You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>
> -- Roberto Viola Software engineer and open source enthusiast http://robertoviola.cloud

Sorry, you mean i can open zwo file on qz on rpi?

**Reply by cagnulein** [reply]
Yes, You can open that files on a raspberry with the -qml Argument in order to have the same interface that you have on the phone
I never tried a zwo File for treadmill so if it doesn't work just attach it here and I will check it. It should be easy to fix in case

**Reply by cagnulein** [reply]
did you try @grigory707 ?

**Reply by nieko-nera** [reply]
Hi, I tried to use ZWO workout file with treadmill, but its not working, here is workout example:
[workout-2256U-economy.zip](https://github.com/cagnulein/qdomyos-zwift/files/7883851/workout-2256U-economy.zip)

**Reply by cagnulein** [reply]
@nieko-nera ok i open an issue about this
